tests/test_georaster.py

```diff
@@ -31,6 +32,64 @@ def test_info(self):
         # Check summary matches that of RIO
         assert print(r) == print(r.info())
 
+    def test_loading(self):
```

Comment:
What do you think about that @rhugonnet ?

Reply:
I think it looks good!